lib/cutest.rb

```diff
@@ -148,19 +148,19 @@ def test(name = nil, &block)
     cutest[:test] = nil
   end
 
-  # Assert that value is not nil or false.
-  def assert(value)
-    flunk("expression returned #{value.inspect}") unless value
+  # Assert that test is a true value.
+  def assert(test)
+    flunk("expression returned #{test.inspect}") unless test
     success
   end
 
-  # Assert that two values are equal.
-  def assert_equal(value, other)
-    flunk("#{value.inspect} != #{other.inspect}") unless value == other
+  # Assert that expected and actual values are equal.
+  def assert_equal(expected, actual)
```

Comment:
I think @soveran and I use it the other way around, similar to how you usually code an `if`:

```
if foo == 2
```

```
assert_equal foo, 2
```

Right?